static/css/devhub/new-landing/base.less

```diff
@@ -3,11 +3,11 @@
 
 // Grid
 @grid-max: 940px;
-@extra-small: ~"only screen and (max-width: 599px)";
-@small: ~"only screen and (min-width: 599px)";
+@extra-small: ~"only screen and (max-width: 399px)";
+@small: ~"only screen and (min-width: 400px)";
 @medium: ~"only screen and (min-width: 700px)";
 @large: ~"only screen and (min-width: 1024px)";
-@retina: ~"only screen and (min-device-pixel-ratio: 2)";
+@retina: ~"only screen and (min-resolution: 1.5dppx)";
```

Comment:
This is the proper way to do resolution queries since min-device-pixel-ratio is old hat.